eclass/unpacker.eclass

```diff
@@ -604,7 +618,7 @@ unpacker_src_uri_depends() {
 		*.zip)
 			d="app-arch/unzip" ;;
 		*.lz)
-			d="|| ( app-arch/plzip app-arch/pdlzip app-arch/lzip )" ;;
+			d="|| ( >=app-arch/xz-utils-5.4.0 app-arch/plzip app-arch/pdlzip app-arch/lzip )" ;;
```

Comment:
Please make it multiline.

Reply:
Changed indentation - not sure if I like how it looks though.